templates/workspaces/base/terraform/locals.tf

```diff
@@ -0,0 +1,6 @@
+locals {
+  tre_core_tags = {
```

Comment:
This isn't "core", please rename to **tre_workspace_tags** and use in all the places.